### DISCUSSION TITLE
Hora de revisión exámenes

### DISCUSSION THREAD

**Discussion by andreslb9** [mmasias/23-24-prg1#175] [Q&A]
Hola, al final la hora de la revisión de los exámenes es a las 11 mañana miércoles no?

**Comment by mmasias** [comment]
No: en cuanto termine de corregir y publique la nota definitiva se los haré saber: estimo que será en el transcurso de esta tarde o por la noche.

**Reply by andreslb9** [reply]
De acuerdo
